<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeWiseGUIClass</class>
 <widget class="QMainWindow" name="TimeWiseGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>802</width>
    <height>436</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>802</width>
    <height>436</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TimeWiseGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>601</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>20</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;input commands here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>+ Add Task</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_task">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
font: 75 italic 42pt &quot;Christopherhand&quot;;</string>
    </property>
    <property name="text">
     <string>Your Tasks</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mlog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to TiMEWiSE!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_help">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>add &lt;task description&gt;&lt;duedate&gt;&lt;category&gt;&lt;priority&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_desc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Rockwell Condensed&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>What would you like to do today?</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>781</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/TimeWiseGUI/123.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
font: 75 26pt &quot;Homestead Display&quot;;</string>
    </property>
    <property name="text">
     <string>SAT 20 feb 2014</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Homestead Display&quot;;
color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>13:20:20</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="timewisegui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>TimeWiseGUIClass</receiver>
   <slot>lineEditReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>lineEditReturnPressed()</slot>
 </slots>
</ui>
